conf/zapiperf/7mode/8.2.5/copy_manager.yaml

```diff
@@ -14,7 +14,7 @@ counters:
   - KB_copied               => kb_copied
 
 override:
- - KB_copied: delta
```

Comment:
can u also check on cmode? Could you also run yaml validator on these yamls if it can identify the issue we saw in logs?

Reply:
Yes, yaml validator can identify the indentation issue. cmode yamls are fine.

hardikl@hardikl-mac-0 harvest % yamllint conf/zapiperf/7mode/8.2.5/copy_manager.yaml 
conf/zapiperf/7mode/8.2.5/copy_manager.yaml
  1:1       warning  missing document start "---"  (document-start)
  1:26      error    too many spaces after colon  (colons)
  2:26      error    too many spaces after colon  (colons)
  3:26      error    too many spaces after colon  (colons)
  5:26      error    too many spaces after colon  (colons)
  17:2      error    wrong indentation: expected 2 but found 1  (indentation)

Ran for all yamls, but the result is little different.
example: 
for zapi's field and sub field, it's recognising the issue but we didn't get any issue from this.
![image](https://user-images.githubusercontent.com/83282894/135295156-2fdd6246-fb19-4cde-bc1d-026bbf059665.png)

Reply:
use `yamllint -d relaxed` to ignore the superfluous messages